packages/lesswrong/server/migrations/20231205T204412.make_fields_not_nullable.ts

```diff
@@ -740,8 +740,7 @@ const setNotnullCommands = `
     ALTER COLUMN "userId" SET NOT NULL;
 
   ALTER TABLE "Images"
-    ALTER COLUMN "cdnHostedUrl" SET NOT NULL,
-    ALTER COLUMN "originalUrl" SET NOT NULL;
+    ALTER COLUMN "cdnHostedUrl" SET NOT NULL;
```

Comment:
Is the nullability of `originalUrl` changing?  I don't see any corresponding changes in the schema (or the new migration)